scripts/update-javascript.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+
+tagPrefix=''
+repo="https://github.com/getsentry/sentry-javascript.git"
+packages=('@sentry/browser' '@sentry/core' '@sentry/integrations' '@sentry/react' '@sentry/vue' '@sentry/angular' '@sentry/angular-ivy' '@sentry/types' '@sentry/utils')
```

Comment:
I believe this won't work for `@sentry/angular-ivy`, `@sentry/angular`, `@sentry/react`, and `@sentry/vue`. As the `update-package-json.sh` runs `yarn upgrade`. 

I tried to run `yarn upgrade` and it added these as deps not upgrading the peers.

Reply:
nice catch!, sucks that yarn upgrade doesn't support peerDependencies...
I added another way of updating the peerDependencies by using the add command with the --peer flag instead of relying on the upgrade command